src/promptflow/promptflow/_sdk/_serving/app.py

```diff
@@ -179,6 +179,7 @@ def version():
 
 def create_app(**kwargs):
     app = PromptflowServingApp(__name__)
+    print("create_app: ", __name__)
```

Comment:
user logging instead of print?

Reply:
sorry, this is for local debug, let me remove this.